src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.main-content {
+  &__catalog {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 944px;
```

Comment:
13.[LAYOUT] - Use sass functions for dynamic max-width calculation, your catalog container should work fine for any number of cards.
Sorry=)

Reply:
thanks, fixed